src/app.js

```diff
@@ -1 +1,44 @@
+/* eslint-disable no-shadow */
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = (source, destination) => {
+  if (!source || !destination) {
+    throw new Error('Source and destination must be provided');
+  }
+
+  if (source === destination) {
+    throw new Error('Source and destination paths are the same');
+  }
+
+  if (!fs.existsSync(source)) {
+    throw new Error(`${source} doesn't exist`);
```

Comment:
Consider providing more information in the error message, such as suggesting the user check the file path.